src/request/shard.rs

```diff
@@ -30,21 +36,27 @@ pub trait Shardable {
     type Shard: Send;
 
     fn shards(
```

Comment:
I think there can be a better name like `group_shards_by_store`

Reply:
I'd like to make a following PR to refactor these parts.